llvm/include/llvm/Analysis/CtxProfAnalysis.h

```diff
@@ -56,6 +57,10 @@ class PGOContextualProfile {
 
   const PGOCtxProfile &profiles() const { return Profiles; }
 
+  // True if this module is a post-thinlto module containing just functions
+  // participating in one or more contextual profiles.
```

Comment:
All the other comments are on the members instead of the accessor. Lets move this and make it consistent?